docs/products/clickhouse/get-started.md

```diff
@@ -5,77 +5,483 @@ sidebar_label: Get started
-    **ClickHouse database**.
+   ```go
+   kubectl apply -f clickhouse-sample.yaml
+   ```
 
-1.  In the **Create ClickHouse database** window, enter a name for your
-    database and select **Create database**.
+1. Review the resource you created with the following command:
 
+   ```go
+   kubectl get clickhouse.aiven.io clickhouse-sample
+   ```
+
+The output is similar to the following:
+
+```text
+NAME                PROJECT        REGION                PLAN        STATE
+clickhouse-sample   your-project   google-europe-west10  startup-16  RUNNING
+```
+
+The resource can stay in the `REBUILDING` state for a couple of minutes. Once the state
+changes to `RUNNING`, you are ready to access it.
 </TabItem>
-<TabItem value="2" label="Aiven API">
+</Tabs>
+
+## Configure the service
+
+Edit your service settings if the default service configuration doesn't meet your needs.
+
+<Tabs groupId="group1">
+<TabItem value="1" label="Console" default>
+1. Select the new service from the list of services on
+   the <ConsoleLabel name="Services"/> page.
+1. On the <ConsoleLabel name="overview"/> page, select <ConsoleLabel name="service settings"/>
+   from the sidebar.
+1. In the **Advanced configuration** section, make changes to the service configuration.
+</TabItem>
+<TabItem value="2" label="Terraform">
+
+Configure service parameters by updating the `aiven_clickhouse` resource, for example:
+
+```hcl
+resource "aiven_clickhouse" "clickhouse" {
+  project      = var.project_name
+  service_name = var.service_name
+  cloud_name   = var.cloud_name
+  plan         = var.service_plan
++
++  maintenance_window_dow  = "monday"
++  maintenance_window_time = "01:00:00"
++  termination_protection  = true
++
++  clickhouse_user_config {
++    ip_filter_string = ["10.20.0.0/16"]
++    service_log      = true
++  }
++
++  clickhouse {
++    uris = [Clickhouse-server-URIs]
++  }
+}
+
+output "clickhouse_service_host" {
+  value = aiven_clickhouse.clickhouse.service_host
+}
+
+output "clickhouse_service_port" {
+  value = aiven_clickhouse.clickhouse.service_port
+}
+
+output "clickhouse_service_username" {
+  value = aiven_clickhouse.clickhouse.service_username
+}
+
+output "clickhouse_service_password" {
+  value     = aiven_clickhouse.clickhouse.service_password
+  sensitive = true
+}
+```
+
+</TabItem>
+<TabItem value="3" label="K8s">
+1. Update file `clickhouse-sample.yaml` with the following content:
+
+   ```yaml
+   apiVersion: aiven.io/v1alpha1
+   kind: ClickHouse
+   metadata:
+   name: clickhouse-sample
+   spec:
+   authSecretRef:
+       name: aiven-token
+       key: token
 
-Call the
-[ServiceClickHouseDatabaseCreate](https://api.aiven.io/doc/#tag/Service:_ClickHouse/operation/ServiceClickHouseDatabaseCreate)
-endpoint to create a database.
+   connInfoSecretTarget:
+       name: clickhouse-connection
 
+   project: <your-project-name>
+
+   cloudName: google-europe-west12
+   plan: business-16
```

Comment:
The plan and cloud fields are immutable.

Reply:
I see. @byashimov could you please give examples of mutable fields I can use instead?